connexion/decorators/uri_parsing.py

```diff
@@ -115,8 +133,21 @@ def resolve_params(self, params, _in):
             else:
                 resolved_param[k] = values[-1]
 
+        # set defaults if values have not been set yet
+        resolved_param = self.set_default_values(resolved_param, self.param_schemas)
```

Comment:
maybe this should go with the other defaults code?
( https://github.com/zalando/connexion/blob/master/connexion/operations/openapi.py#L308 )